src/App/Services/PushNotificationListenerService.cs

```diff
@@ -131,6 +131,24 @@ public async Task OnMessageAsync(JObject value, string deviceType)
                         _messagingService.Value.Send("logout");
                     }
                     break;
+                case NotificationType.SyncSendCreate:
+                case NotificationType.SyncSendUpdate:
+                    var sendCreateUpdateMessage = JsonConvert.DeserializeObject<SyncSendNotification>(
+                        notification.Payload);
+                    if (isAuthenticated && sendCreateUpdateMessage.UserId == myUserId)
+                    {
+                        await _syncService.Value.SyncUpsertSendAsync(sendCreateUpdateMessage,
+                            notification.Type == NotificationType.SyncCipherUpdate);
```

Comment:
```suggestion
                            notification.Type == NotificationType.SyncSendUpdate);
```
Should be `NotificationType.SyncSendUpdate` instead `NotificationType.SyncCipherUpdate`